pkgs/top-level/python-packages.nix

```diff
@@ -7282,6 +7314,26 @@ let
 
   };
 
+  gnureadline = buildPythonPackage rec {
+    version = "6.3.3";
+    name = "gnureadline-${version}";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/g/gnureadline/gnureadline-6.3.3.tar.gz";
+      md5 = "c4af83c9a3fbeac8f2da9b5a7c60e51c";
+    };
+
+    propagatedBuildInputs = with self; [];
+
+    meta = {
+      description = "Stand-alone GNU readline module for Python ";
+      homepage = https://github.com/ludwigschwardt/python-gnureadline;
+      license = licenses.bsd3;
+      maintainers = with maintainers; [ fridh ];
+    };
```

Comment:
I don't think I am the maintainer here? You could add yourself here if you would like to maintain this :-)